src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -84,12 +84,27 @@ system environment variables with the same name.
 
 ## Environment variables k6 will read automatically
 
-k6 will also try to read some specific environment variables that the user can set to change the
-behavior of k6:
+k6 will read some specific environment variables to customize its operation.
+These must be supplied to k6 at the shell level (so they _cannot_ be [passed
+via -e or --env][envvarsref]).
 
 | Name                 | Description                                                                                                            |
 | -------------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `K6_CLOUD_HOST`      | A URL to connect to, when the --out=cloud results output option is specified.                                          |
-| `K6_CLOUD_TOKEN`     | An authentication token to use in API calls to the cloud service, when the --cloud results output option is specified. |
+| `K6_CLOUD_HOST`      | A URL to connect to when the `--out=cloud` results output option is specified.                                          |
+| `K6_CLOUD_TOKEN`     | An authentication token to use in API calls to the k6 cloud service, both for [streaming results to][k6ingest] and for [running tests in](/cloud) the k6 cloud. |
 | `K6_NO_USAGE_REPORT` | Define this to disable [usage reports](/misc/usage-collection).                                                        |
 | `K6_OUT`             | Specify results output, same as --out command-line option.                                                             |
+
+k6 also allows proxying via its use of [Golang's standard http library](https://pkg.go.dev/net/http#ProxyFromEnvironment).
+This is useful e.g. when wanting to send data to [k6 cloud](/cloud) from a k6 running inside a restricted network.
+
+| Name                 | Description                                                          |
+| -------------------- | ---------------------------------------------------------------------|
+| `HTTP_PROXY`         | Proxy address (URL or `host:port`) to use for HTTP requests, unless overridden by `NO_PROXY`.    |
+| `HTTPS_PROXY`        | Proxy address (URL or `host:port`) to use for HTTPS requests, unless overridden by `NO_PROXY`.   |
+| `NO_PROXY`           | Hosts that should bypass proxies. This is a comma-separated string of IP addresses and/or host names. See [the httpproxy docs][httpproxyconfig] for more info. Example: `"1.2.3.4, 2.3.4.5:80, myhost.com"`.|
```

Comment:
not sure about the spaces in the example :thinking:

Reply:
Fair enough, I removed them to be on the safe side. :sweat_smile: